### PR TITLE
Added types for wordpos 1.1

### PR DESCRIPTION
**Pull request by guidsdo**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Review by plantain-00 — CHANGES_REQUESTED** [review]
A test file is required

**Comment by guidojo** [comment]
@plantain-00 added tests, thought they weren't required in that manner. Had to rebase as well because author was incorrect

**Review by plantain-00 — APPROVED** [review]

File: types/wordpos/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+
+    /**
+     * Get deduped, less stopwords
+     *
+     * @param text string of words to parse.  If array is given, it is left in tact.
+     */
+    parse(text: string | string[]): string[];
+}
+
+export default WordPOS;
```
Comment (armanio123) [inline types/wordpos/index.d.ts]: The library doesn't have a default export. Please change to use "export ="

File: types/wordpos/wordpos-tests.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
+import WordPOS from 'wordpos';
```
Comment (armanio123) [inline types/wordpos/wordpos-tests.ts]: Use import x = require("module") instead, as it doesn't define a default.

**Review by armanio123 — CHANGES_REQUESTED** [review]

**Comment by mhegazy** [comment]
Thanks for your contribution. This PR has unaddressed comments and can not be merged in at the time being. For housekeeping purposes we are closing stale PRs. If you'd still like to merge this code in, please open a new PR that has been merged and rebased with the `master` branch.

**Pull request closed without merging**